src/main/default/lwc/createCaseScreen/createCaseScreen.js

```diff
@@ -90,7 +91,12 @@ export default class CreateCaseScreen extends LightningElement {
 
   connectedCallback() {
     registerComponentForInit(this, this.engineId);
-    this.extractDataFromSessionStorage();
+    try {
+      this._caseData = JSON.parse(this.caseData);
+      this.extractDataFromSessionStorage();
+    } catch (err) {
```

Comment:
I guess we should, at least, log a warning. I assume `this.caseData` might be `undefined` under some circumstances. You might want to validate that before calling `JSON.parse()` and log only the "real" errors.